README.npm.md

```diff
@@ -0,0 +1,200 @@
+[![Documentation](https://readthedocs.org/projects/kedro/badge/?version=stable)](https://docs.kedro.org/en/stable/visualisation/)
+[![Python Version](https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11-orange.svg)](https://pypi.org/project/kedro-viz/)
+[![PyPI version](https://img.shields.io/pypi/v/kedro-viz.svg?color=yellow)](https://pypi.org/project/kedro-viz/)
+[![Downloads](https://static.pepy.tech/badge/kedro-viz/week)](https://pepy.tech/project/kedro-viz)
+[![npm version](https://img.shields.io/npm/v/@quantumblack/kedro-viz.svg?color=cc3534)](https://badge.fury.io/js/%40quantumblack%2Fkedro-viz)
+[![License](https://img.shields.io/badge/license-Apache%202.0-3da639.svg)](https://opensource.org/licenses/Apache-2.0)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+[![Slack Organisation](https://img.shields.io/badge/slack-chat-blueviolet.svg?label=Kedro%20Slack&logo=slack)](https://slack.kedro.org)
+
+## Introduction
+
+Kedro-Viz is an interactive development tool for building data science pipelines with [Kedro](https://github.com/kedro-org/kedro). Kedro-Viz also allows users to view and compare different runs in the Kedro project.
+
+## Features
+
+- ✨ Complete visualisation of a Kedro project and its pipelines
+- 🎨 Supports light & dark themes out of the box
+- 🚀 Scales to big pipelines with hundreds of nodes
+- 🔎 Highly interactive, filterable and searchable
+- 🔬 Focus mode for modular pipeline visualisation
+- 📊 Rich metadata side panel to display parameters, plots, etc.
+- 📊 Supports all types of [Plotly charts](https://plotly.com/javascript/)
+- ♻️ Autoreload on code change
+- 🧪 Supports tracking and comparing runs in a Kedro project
+- 🎩 Many more to come
+
+## Installation
+
+- As a standalone React component (for embedding Kedro-Viz in your web application).
+
+  To install the standalone React component:
+
+  ```bash
+  npm install @quantumblack/kedro-viz
+  ```
+
+## Usage
+
+To use Kedro-Viz as a standalone React component, you can follow the example below. However, please note that Kedro-Viz does not support server-side rendering (SSR). If you're using Next.js or another SSR framework, you should be aware of this limitation.
+
+```javascript
+import KedroViz from '@quantumblack/kedro-viz';
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+
+const MyApp = () => <KedroViz data={json} />;
+```
+
+To use with NextJS:
+
+```javascript
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+import dynamic from 'next/dynamic';
+
+const NoSSRKedro = dynamic(() => import('@quantumblack/kedro-viz'), {
+  ssr: false,
+});
+
+const MyApp = () => <NoSSRKedro data={json} />;
+```
+
+The JSON can be obtained by running:
+
+```bash
+kedro viz run --save-file=filename
+```
+
+The command will generate a folder with the specified <filename>, containing the following structure: 
```

Comment:
```suggestion
On successful execution of the command above, it will generate a folder at the specified directory, containing the following structure: 
```